docs/UserGuide.adoc

```diff
@@ -221,6 +231,7 @@ The `redo` command fails as there are no `undo` commands executed previously.
 === Clearing all entries : `clear`
 
 Clears all entries from the address book. +
+Alias: `c` +
 Format: `clear`
 
 === Exiting the program : `exit`
```

Comment:
Nice to think about updating user guide (although it seems like only one person remembered...)